pkg/remote/utils.go

```diff
@@ -0,0 +1,286 @@
+package remote
```

Comment:
Note that "remote" is not an ideal name. In eden we use "remote" to denote physical edge node controlled by eden over the network - as opposed to locally started VM running EVE. However, this package I suspect is mostly going to be used for testing EVE running in a VM.

Reply:
yes, this is more of auxiliary package for testing stuff on EVE and VMs, what do you think would be a good name?